neighbour-model/src/main/java/no/vegvesen/ixn/federation/transformer/SubscriptionRequestTransformer.java

```diff
@@ -45,11 +45,27 @@ public SubscriptionRequest subscriptionRequestApiToSubscriptionRequest(Subscript
 	}
 
 	public Subscription subscriptionPollApiToSubscription(SubscriptionPollResponseApi subscriptionApi) {
-		Subscription subscription = new Subscription(subscriptionApi.getSelector(), subscriptionTransformer.subscriptionStatusApiToSubscriptionStatus(subscriptionApi.getStatus()));
-		subscription.setPath(subscriptionApi.getPath());
-		subscription.setBrokerUrl(subscriptionApi.getMessageBrokerUrl());
-		subscription.setQueue(subscriptionApi.getQueueName());
-		return subscription;
+		if (subscriptionApi.isCreateNewQueue() == null) {
```

Comment:
isCreateNewQueue can potentially return Boolean::false, so you have test for false in the if, as well